lib/perl/Genome/Model/CwlPipeline/Command/Run.pm

```diff
@@ -227,90 +227,93 @@ sub run_cromwell_gcp {
 
     my $poll_interval_seconds = 300;
 
-    #
-    # Cloudize workflow
-    #
-    my $cloud_yaml = $yaml; $cloud_yaml =~ s/.ya?ml/_cloud.json/;
     my $lsb_sub_guard = Genome::Config::set_env('lsb_sub_additional', 'docker(mgibio/cloudize-workflow:1.2.2)');
     delete local $ENV{BOTO_CONFIG};
 
-    Genome::Sys::LSF::bsub::bsub(
-        queue => $queue,
-        user_group => $user_group,
-        resource_string => 'rusage[mem=512M:internet2_upload_mbps=500]',
-        wait_for_completion => 1,
-        log_file => File::Spec->join($logdir, '00_cloudize_workflow.log'),
-        cmd => [
-            "python3",
-            "/opt/scripts/cloudize-workflow.py",
-            $bucket,
-            $main_workflow_file,
-            $yaml,
-            "--output=$cloud_yaml"] );
-
-    # Zip dependencies
-    my $deps_zip_path = File::Spec->join($data_dir, 'deps.zip');
-    my $deps_zip_url = "gs://$bucket/build.$build_id/deps.zip";
-    my $prev_dir = getcwd;
-    my(undef, $deps_dir, undef) = File::Spec->splitpath($main_workflow_file);
+    if ($self->_cloud_instance_is_running) {
+        $self->status_message("Detected already running cloud VM for job.  Will monitor for completioni every $poll_interval_seconds seconds.");
```

Comment:
s/completioni/completion/

Reply:
Oops, good old "typing `i` while already in insert mode"...